CHANGES.txt

```diff
@@ -49,6 +49,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       cl.exe-compatible command line switches.
     - Some manpage cleanup for the gettext and pdf/ps builders.
     - Some clarifications in the User Guide "Environments" chapter.
+    - Fixed wrong naming in the documentation of "keep_local" parameter for 
```

Comment:
This should be changed to be attributed to you and not to mwichmann.
Please add a section with your name here ( it should be alphabetical by last name)